errors.go

```diff
@@ -499,3 +545,30 @@ func (e *RFC6749Error) ToValues() url.Values {
 
 	return values
 }
+
+func (e *RFC6749Error) getDefaultHint(ID i18n.ErrorHintType, args ...interface{}) string {
+	defHint, ok := i18n.HintsMap[ID]
+	if !ok {
+		// This should never happen if the hints are properly populated in the
+		// map.
+		panic(fmt.Sprintf("Hint must be populated in the map for: %s", ID))
```

Comment:
I would much rather prefer not to panic here but instead return a generic error message

Reply:
Addessed